packager/media/codecs/nal_unit_to_byte_stream_converter_unittest.cc

```diff
@@ -80,24 +80,58 @@ TEST(NalUnitToByteStreamConverterTest, EmptyAVCDecoderConfigurationRecord) {
   EXPECT_FALSE(converter.Initialize(nullptr, 102));
   EXPECT_FALSE(converter.Initialize(kTestAVCDecoderConfigurationRecord, 0));
 }
-
-// If there is no SPS, Initialize() should fail.
-TEST(NalUnitToByteStreamConverterTest, NoSps) {
+        
+// Expect a valid AVCDecoderConfigurationRecord with SPSExtension to pass.
+TEST(NalUnitToByteStreamConverterTest, SpsWithExtension) {
```

Comment:
Can you move it after `ConvertUnitToByteStream` test? And we can call it `ConvertUnitToByteStreamWithSpsExtension`.

Reply:
done, name change and moved